tests/test_processors.py

```diff
@@ -850,12 +844,17 @@ def __init__(self):
 
         logger = structlog.stdlib.get_logger()
 
+        # These are different when running under async
+        exclude_keys = ["thread", "thread_name"]
+
```

Comment:
This can be removed I think (see below).

Reply:
Yeap, killed with the list context pop change.